code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -20,6 +20,13 @@
 	. = ..()
 	if(HAS_TRAIT(src, TRAIT_DEAF))
 		return INFINITY //For all my homies that can not hear in the world
+
+	var/turf/current_turf = get_turf(src)
+	var/datum/gas_mixture/environment = current_turf.return_air()
+	var/pressure = environment ? environment.return_pressure() : 0
+	if(pressure < SOUND_MINIMUM_PRESSURE) //space is empty
+		return INFINITY
```

Comment:
I have feeling these factors shouldn't be exclusive to carbon mobs, so I suggest you move these to `/mob/living/proc/get_ear_protection()`.

Reply:
sure

Reply:
done